hw/ip/aon_timer/doc/_index.md

```diff
@@ -59,6 +60,9 @@ The "pause in sleep" option controls whether the watchdog timer continues to cou
 This allows configurations where the watchdog timer can remain programmed and locked while the device is put to sleep for relatively long periods, controlled by the wakeup timer.
 Without this feature, the watchdog timer might wake up the core prematurely by triggering a watchdog bark.
 
+The "pause during escalation" feature ensures that watchdog bites and barks do not interfere with system escalation behavior.
+If the system during escalation decides to hang instead of resetting, the watchdog bite cannot supersede that decision.
```

Comment:
```suggestion
If during escalation the system hangs instead of resetting, the watchdog bite cannot supersede that behavior.
```
The system doesn't "decide", the designer does; the system just operates :)

Reply:
eh? this case really is decide. whether we hang during escalation is a software configuration.

Reply:
```suggestion
If during escalation the system decides to hang instead of resetting, the watchdog bite cannot supersede that decision.
```
This suggestion is just for slightly better grammar.

Reply:
Okay, the system decides :)

Reply:
o that wording is better, thanks for the suggestion.